releasenotes/notes/0.13/support-scheduler-in-execute-42a1ebe9e3a67bbb.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    :func:`qiskit.execute.execute` has a new kwarg ``schedule_circuit``. By
```

Comment:
```suggestion
    :func:`qiskit.execute` has a new kwarg ``schedule_circuit``. By
```

Reply:
This is actually correct as is. It's the doc path which comes from the python path of the object. In this case it's `qiskit.execute.execute`: https://qiskit.org/documentation/apidoc/execute.html

Reply:
maybe we should replace this with `~qiskit` so it just shows `execute()` instead of `qiskit.execute.execute`?

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/4057/commits/31da6c99ba275c899eaf6f8483d5edac2b596efb